<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <node name="drl_training" pkg="transfer_learning" type="drl_training" output="screen"/>
</launch>
